platform/kobo/koreader.sh

```diff
@@ -114,9 +114,20 @@ fi
 # we keep maximum 500K worth of crash log
 cat crash.log 2> /dev/null | tail -c 500000 > crash.log.new
 mv -f crash.log.new crash.log
+
+# TODO: add support for 32bit framebuffer in koreader. This is a workaround
+# to run koreader in 16bpp. Useful since FW 4.0
+FB_BPP=$(cat /sys/class/graphics/fb0/bits_per_pixel)
+[ "$FB_BPP" -gt 16 ] && /usr/sbin/fbset -depth 16
+# fix rotation issues on Kobo Aura HD (dragon) and Kobo Aura H2O (dahlia)
```

Comment:
From KSM this actually "cause". There was some stuff about this on https://github.com/koreader/koreader/commit/c2bd85bba40e4c6e741af7321c73685e32bfa555#commitcomment-12846700